src/__myproject__/_export/p360_export.py

```diff
@@ -0,0 +1,20 @@
+# Databricks notebook source
+# MAGIC %run ../app/bootstrap
+
+# COMMAND ----------
+
+import daipe as dp
+from p360_export.P360ExportRunner import P360ExportRunner
+
+# COMMAND ----------
+
+@dp.notebook_function()
+def init_widgets(widgets: dp.Widgets):
+    widgets.add_text("config_url", "", "Config url")
+
+# COMMAND ----------
+
+@dp.notebook_function()
```

Comment:
use `dp.fs.get_widget_value("config_url")` in decorator instead of `config_url = widgets.get_value("config_url")` inside the funciton